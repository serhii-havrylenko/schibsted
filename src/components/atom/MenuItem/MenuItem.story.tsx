import { storiesOf } from '@storybook/react';
import * as React from 'react';

import MenuItem from './MenuItem';

storiesOf('MenuItem', module).add('default', () => <MenuItem>Home</MenuItem>);
