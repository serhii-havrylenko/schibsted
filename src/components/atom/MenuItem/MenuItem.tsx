import Typography from '@material-ui/core/Typography';
import * as React from 'react';
import styled from 'styled-components';

const MenuItem: React.SFC<{ className?: string }> = ({
  className,
  children,
}) => (
  <Typography variant="h6" color="inherit" noWrap={false} className={className}>
    {children}
  </Typography>
);

export default styled(MenuItem)`
  && {
    color: #fff;
  }
`;
