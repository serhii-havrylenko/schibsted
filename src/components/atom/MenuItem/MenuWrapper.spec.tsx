import { mount } from 'enzyme';
import * as React from 'react';

import MenuItem from './MenuItem';

jest.mock('@material-ui/core/Typography');

describe('<MenuItem /> tests', () => {
  test('should match snapshot', () => {
    const tree = mount(<MenuItem />);
    expect(tree).toMatchSnapshot();
  });
});
