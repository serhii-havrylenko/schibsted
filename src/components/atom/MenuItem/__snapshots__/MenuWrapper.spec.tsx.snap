// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> tests should match snapshot 1`] = `
.c0.c0 {
  color: #fff;
}

<Styled(MenuItem)>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
  && {
    color: #fff;
  }
",
          ],
        },
        "displayName": "Styled(MenuItem)",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": [Function],
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <MenuItem
      className="c0"
    >
      <WithStyles(Typography)
        className="c0"
        color="inherit"
        noWrap={false}
        variant="h6"
      />
    </MenuItem>
  </StyledComponent>
</Styled(MenuItem)>
`;
